visualization/.eslintrc.js

```diff
@@ -1,86 +1,87 @@
 module.exports = {
```

Comment:
It seems like nothing changed here. Was this a formatting/linting mistake? or was a specific rule changes?

Reply:
I added `"unicorn/no-useless-undefined": "off"`. Otherwise e.g. `function foo(a: string | undefined)` is always unhappy as `foo()` is missing one parameter and `foo(undefined)` has an "usless" undefined. In my strong opinion second call is the correct way, so I disabled this rule.